### PR TITLE
[MNOE-597][WIP] Missing Conversion

### PR DESCRIPTION
**Pull request by adamaziz15**
@alexnoox Still WIP, I was unsure about the best way to translate the roles. The way I currently have it works, I tried using pluralisation but was having an issue with messageformat throwing locale errors. Please review that part when you have a chance. :)

### CODE REVIEW

File: src/app/views/company/members/organization-members.directive.coffee [hunk 1]
```diff
@@ -46,20 +46,21 @@ DashboardOrganizationMembersCtrl = ($scope, $uibModal, $sce, MnoeOrganizations,
 
 
   $scope.memberRoleLabel = (member) ->
-    if member.entity == 'User'
-      return member.role
-    else
-      return "Invited (#{member.role})"
+    invited = if member.entity == 'User' then "" else "invited."
+    "mno_enterprise.templates.dashboard.organization.members.roles." + invited + member.role.split(" ").join("_").toLowerCase()
```
Comment (alexnoox) [inline src/app/views/company/members/organization-members.directive.coffee]: You can use lodash https://lodash.com/docs/3.10.1#snakeCase

File: src/app/views/company/members/organization-members.directive.coffee [hunk 2]
```diff
@@ -178,7 +179,13 @@ DashboardOrganizationMembersCtrl = ($scope, $uibModal, $sce, MnoeOrganizations,
     roles: ->
       list = ['Member','Admin']
       list.push('Super Admin') if MnoeOrganizations.role.isSuperAdmin()
-      return list
+      translated_list = []
+      _.each(list, (role) -> $translate("mno_enterprise.templates.dashboard.organization.members.roles." + role.split(" ").join("_").toLowerCase()).then(
```
Comment (alexnoox) [inline src/app/views/company/members/organization-members.directive.coffee]: You can use lodash https://lodash.com/docs/3.10.1#snakeCase

File: src/app/views/company/members/organization-members.directive.coffee [hunk 3]
```diff
@@ -178,7 +179,13 @@ DashboardOrganizationMembersCtrl = ($scope, $uibModal, $sce, MnoeOrganizations,
     roles: ->
       list = ['Member','Admin']
```
Comment (alexnoox) [inline src/app/views/company/members/organization-members.directive.coffee]: Why don't you simply pass the list of roles locale keys?

**Comment by ouranos** [comment]
@adamaziz15 Merging. Could you open another PR to apply @alexnoox recommendations?

**Pull request merged**